.github/workflows/ci.yml

```diff
@@ -69,6 +83,12 @@ jobs:
         with:
           suite: ${{ matrix.suite }}
           os: ${{ matrix.os }}
+      - name: Print debug output on failure
+        if: failure()
+        run: |
+            set -x
+            sudo journalctl -l --since today
+            KITCHEN_LOCAL_YAML=kitchen.dokken.yml /usr/bin/kitchen exec ${{ matrix.suite }}-${{ matrix.os }} -c "journalctl -l"
```

Comment:
if we keep doing this everywhere, why don't we extend the github action to do this by default?

Reply:
Should I keep this here for now and work on a separate PR which adds this to the action?

Reply:
yeah, I think that is best.